test/sandbox-test.js

```diff
@@ -113,6 +113,13 @@ describe("sinon.sandbox", function () {
                     assert(sinon.useFakeXMLHttpRequest.called);
                 });
 
+                it("doesn't secretly use useFakeServer", function () {
+                    this.sandbox.stub(sinon.fakeServer, "create").returns({ restore: function () {} });
```

Comment:
Going through old PRs that have stalled to try and get them rolling again ... 

@fearphage : other than simply exchanging `sinon.fakeServer` with `sinon.fakeServerWithClock` to (hopefully) get a test that failed prior to the change, do you have a better way of asserting we are doing the right thing?

@bouk: anything preventing you from changing the test to actually testing the change? The test should fail before your change. I am assuming this is not the case (? - feel free to prove otherwise).